tests/test_math/test_backend_manager.py

```diff
@@ -53,7 +61,8 @@ def test_get_backend(self):
         Tests the ``get_backend`` method.
         """
         assert math.get_backend("numpy").name == "numpy"
-        assert math.get_backend("jax").name == "jax"
+        with contextlib.suppress(ImportError):
+            assert math.get_backend("jax").name == "jax"
```

Comment:
Why is there an `ImportError` here?

Reply:
Should maybe split into `test_get_backend_numpy` and `test_get_backend_jax`, decorating the second with `@pytest.mark.requires_backend("jax")`?